blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubEnterpriseApiTest.java

```diff
@@ -0,0 +1,106 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.google.common.collect.ImmutableMap;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import io.jenkins.blueocean.credential.CredentialsUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * @author cliffmeyers
+ */
+public class GithubEnterpriseApiTest extends GithubMockBase {
+
+    @Test
+    public void validateGithubToken() throws IOException, UnirestException {
+        String credentialId = createGithubEnterpriseCredential();
+        assertEquals("github-enterprise:" + getGithubApiUrlEncoded(), credentialId);
+
+        //check if this credentialId is created in correct user domain
+        Domain domain = CredentialsUtils.findDomain(credentialId, user);
+        assertEquals("blueocean-github-enterprise-domain", domain.getName());
+    }
+
+    @Test
+    public void validateGithubToken_apiUrlRequired() throws UnirestException {
```

Comment:
Method with underscore? Some like using test names with underscore for clarity but it can be achieved with capitalization and words to describe the test. Lets be consistent and keep it capitalized like all other tests.